index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books Project</title>
+  <link rel="stylesheet" href="index.css"> 
+</head>
+<body>
+  <header>
+    <div>Awesome Books</div>
+    <div>
+      <ul id="nav-links">
+        <li> <a href="#allBooks" id="booksLink">List</a> </li>
+        <li> <a href="#book-form" id="formsLink">Add new</a> </li>
+        <li> <a href="#contact" id="contactLink">Contact</a> </li>
+      </ul>
+    </div>
+  </header>
```

Comment:
- [ ] Kindly implement the "luxon" library, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Display%20the%20current%20date%20below%20the%20navigation%20bar.) requirement.